app/views/member/lists/_form.html.haml

```diff
@@ -1,34 +1,32 @@
 = simple_form_for [:member, @list], html: { class: 'form' } do |f|
-  .container
-    .col-md-6
-      %h2 List details
-      .form-group
-        = f.input :name, class: "form-control", label: "Name",
-                  hint: "Give this list a name, for example: Family Secret Santa"
-        = f.input :gift_day, class: "form-control", as: :date
-        .form-inputs
-          = f.input :gift_value, wrapper: :vertical_input_group,
-            hint: "Optional. Does your list have a gift value limit? Set it here." do
-            %span.input-group-addon $
-            = f.input_field :gift_value, class: "form-control",
-              input_html: { min: 0, max: 9999.99 }
-    .col-md-6
-      %h2 Santas
-      #santas
-        .form-group
-          = f.simple_fields_for :santas do |santa|
-            = render "santa_fields", f: santa
-        .links
-          %br
-          = link_to_add_association "Add Santa", f, :santas, class: "btn btn-small btn-success"
+  .col-md-6
+    %h2 List details
+    .form-group
+      -#TODO: Move star on required field to right
```

Comment:
The preceding - can go

Also TODOs can be lost and forgotten easily but I'll leave that to you